source/common/upstream/subset_lb.cc

```diff
@@ -378,11 +378,11 @@ SubsetLoadBalancer::extractSubsetMetadata(const std::set<std::string>& subset_ke
       // Otherwise, we branch the list of kvs by generating one new kvs per old kvs per
       // new value.
       //
-      // For example, two kvs (<a=b>, <c=d>) joined with the kv foo=[bar,baz] results in four kvs:
-      //   <a=b,foo=bar>
-      //   <a=b,foo=baz>
-      //   <c=d,foo=bar>
-      //   <c=d,foo=baz>
+      // For example, two kvs (<a=1>, <a=2>) joined with the kv foo=[bar,baz] results in four kvs:
+      //   <a=1,foo=bar>
+      //   <a=2,foo=baz>
+      //   <a=1,foo=bar>
+      //   <a=2,foo=baz>
```

Comment:
😬I hate to say this, but think the first two are a=1

Reply:
hah no worries, this is what i get for getting sloppy on the tail end of a PR